Mathlib/Topology/Order/UpperLowerSetTopology.lean

```diff
@@ -285,7 +284,7 @@ theorem IsOpen_sInter {S : Set (Set α)} (hf : ∀ s ∈ S, IsOpen s) : IsOpen (
   exact hf _ hs
 
 -- Alexandrov property, index formulation
-theorem isOpen_iInter {f : ι → Set α} (hf : ∀ i, IsOpen (f i)) : IsOpen (⋂ i, f i) := by
+theorem IsOpen_iInter {f : ι → Set α} (hf : ∀ i, IsOpen (f i)) : IsOpen (⋂ i, f i) := by
```

Comment:
This new name is wrong, theorems should be `camelCase_with_underscores`

Reply:
Hopefully fixed now?